packages/advanced-logic/src/extensions/payment-context-exchange-rate.ts

```diff
@@ -0,0 +1,77 @@
+import { ExtensionTypes, RequestLogicTypes } from '@requestnetwork/types';
+
+import Utils from '@requestnetwork/utils';
+
+/**
+ * Implementation of the exchange rate extension
+ */
+const exchangeRate: ExtensionTypes.PcExchangeRate.IExchangeRate = {
+  applyActionToExtension,
+  createCreationAction,
+};
+export default exchangeRate;
+
+const CURRENT_VERSION = '0.1.0';
+
+/**
+ * Creates the extensionsData to create the extension pc-exchange-rate
+ *
+ * @param extensions IAdvancedLogicExtensionsCreationParameters extensions parameters to create
+ *
+ * @returns IExtensionCreationAction the extensionsData to be store in the request
```

Comment:
The type here should be `IAction`.
ps: yes, it seems it's broken on all other extensions :)

Reply:
OK, fixed locally, I'll commit soon.

In the meantime, from `extension-types.ts`:
```
/** Creation action object */
export interface IAction {
  action: string;
  id: ID;
  parameters?: any;
  version?: string;
}
```

`Creation action object` is wrong, isn't it? It should be `Extension action interface`?